c7n/resources/vpc.py

```diff
@@ -471,12 +575,40 @@ class Meta(object):
 
 @NetworkInterface.filter_registry.register('subnet')
 class InterfaceSubnetFilter(net_filters.SubnetFilter):
+    """Network interface subnet filter
+
+    :example:
+
+        .. code-block: yaml
+
+            policies:
+              - name: network-interface-subnet-x
+                resource: eni
+                filters:
+                  - type: subnet
+                    key: SubnetId
+                    value: subnet-01ab23cd
+    """
 
     RelatedIdsExpression = "SubnetId"
 
 
 @NetworkInterface.filter_registry.register('security-group')
 class InterfaceSecurityGroupFilter(net_filters.SecurityGroupFilter):
+    """Network interface security group filter
+
+    :example:
+
+        .. code-block: yaml
+
+            policies:
+              - name: network-interface-sg-x
+                resource: eni
+                filters:
+                  - type: security-group
+                    key: GroupId
```

Comment:
this is a bad example, groupid is an attribute of eni, so a value filter would suffice.